src/test/scala/nl/knaw/dans/api/sword2/ResolveFetchItemsSpec.scala

```diff
@@ -58,31 +58,36 @@ class ResolveFetchItemsSpec extends Sword2Fixture with BagStoreFixture {
   it should "result in a Failure when a required file is missing"  in {
     copyToTargetBagDir(REQUIRED_FILE_MISSING)
     DepositHandler.checkFetchItemUrls(targetBagDir, urlPattern) shouldBe a[Success[_]]
-    DepositHandler.checkBagVirtualValidity(targetBagDir) shouldBe a[Failure[_]]
+    val validity = DepositHandler.checkBagVirtualValidity(targetBagDir)
```

Comment:
@vesaakerman : better use http://doc.scalatest.org/1.9.1/index.html#org.scalatest.Inside to directly check the contents of the `Failure` object: 
```scala
 inside(validity) {
      case Failure(e) =>
        e shouldBe a[InvalidDepositException]
        e.getMessage should include("Bag does not have bagit.txt")
    }
```